alembic/env.py

```diff
@@ -0,0 +1,81 @@
+from logging.config import fileConfig
+
+from sqlalchemy import engine_from_config
+from sqlalchemy import pool
+
+from alembic import context
+
+from cities.models import *
```

Comment:
Using wildcard imports (e.g., `from cities.models import *`) is generally discouraged as it can lead to namespace conflicts and makes it harder to determine which names are present in the namespace. It's better to import only the necessary components explicitly.